frontend/src/components/LeaderBoard.tsx

```diff
@@ -0,0 +1,248 @@
+import { Search,ChevronDown, ChevronUp } from "lucide-react"
+import { useState, useRef, useEffect } from "react"
+import { useLeaderboard, LeaderboardEntry } from "../lib/api";
+
+
+export default function Leaderboard() {
+  const [expandedRows, setExpandedRows] = useState<number[]>([])
+  const [searchQuery, setSearchQuery] = useState<string|null>(null)
+  const [showFeedDropdown, setShowFeedDropdown] = useState<boolean>(false)
+  const [showTimeDropdown, setShowTimeDropdown] = useState<boolean>(false)
+  const [selectedFeed, setSelectedFeed] = useState<string>("All Feed")
+  const [selectedTime, setSelectedTime] = useState<string>("All Time")
+  const { data: leaderboard, isLoading, error } = useLeaderboard();
+
+  const feedDropdownRef = useRef<HTMLDivElement>(null);
+  const timeDropdownRef = useRef<HTMLDivElement>(null);
+
+  const feeds = ["All Feed", "Crypto Grant Wire", "Grants", "NEARWEEK", "Public Goods Club"];
```

Comment:
This should come from config, use /api/config or similiar endpoint rather than hard code